submissionuploads.md

```diff
@@ -0,0 +1,60 @@
+# SubmissionUploads Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+The proposed structure is derived from the configuration options already available in DSpace 6 and below. In the first implementation, a single configuration named *default* is expected as these configurations were set at the site level. Introducing an endpoint to manage a collection of configurations we open the door to future extension where different setups can be used for different kind of submissions (theses, technical reports, journal articles, etc.)
```

Comment:
I think that DSpace 6 provides two "submission uploads" out-of-the box:
* [Upload with embargo](https://github.com/DSpace/DSpace/blob/dspace-6.2/dspace/config/item-submission.xml#L232)
* [Upload withouth embargo](https://github.com/DSpace/DSpace/blob/dspace-6.2/dspace/config/item-submission.xml#L243) 

So maybe we should expose both options here instead of only 1 default.

Reply:
Both configuration are now supported with a single implementation. If the accessConditionOptions is null the upload step behavior as the Upload without embargo. I have added a phrase in the contract about that and also noted in the wiki documentation about the configuration file https://wiki.duraspace.org/display/DSPACE/Configuration+changes+in+the+submission+process